### ISSUE TITLE
Please provide linux builds with musl libc

### ISSUE THREAD

**Issue by sivachandran** [psanford/wormhole-william#109]
Thank you for the project. I use it regularly on multiple machines. But presently the linux builds require a specific libc runtime which is not available in my target hosts. So I have to build with `CGO_ENABLED=1 CC=musl-gcc go build --ldflags '-linkmode external -extldflags=-static'` and use it. 

It would be super helpful for people like me if you can also release linux builds with musl libc.

**Comment by Jacalz** [comment]
Does it work if you build with `CGO_ENABLED=off go build -tags netgo`?. I think that should build entirely without CGO support as another option.

**Comment by sivachandran** [comment]
Yes, building with `CGO_ENABLED=off go build -tags netgo` works too. Thanks!
